### PR TITLE
Add support for the isSystemKeyboardSupported attribute

### PR DESCRIPTION
**Pull request by cabanier**
***
<a href="https://pr-preview.s3.amazonaws.com/cabanier/webxr/pull/1314.html" title="Last updated on Mar 3, 2023, 2:18 AM UTC (bbfdf9a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/webxr/1314/1fb14e7...cabanier:bbfdf9a.html" title="Last updated on Mar 3, 2023, 2:18 AM UTC (bbfdf9a)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -653,6 +654,8 @@ Each {{XRSession}} has a <dfn for=XRSession>set of granted features</dfn>, which
 
 The <dfn attribute for="XRSession">enabledFeatures</dfn> attribute returns the features in the [=XRSession/set of granted features=] as a new array of {{DOMString}}s.
 
+The <dfn attribute for="XRSession">isSystemKeyboardSupported</dfn> attribute indicates that the {{XRSystem}} has the ability to display the system keyboard while the {{XRSession}} is active.
```
Comment (Manishearth) [inline index.bs:657]: probably should list the various conditions under which this may happen

Comment (cabanier) [reply index.bs:657]: Do you know where this is called out in the HTML spec? https://html.spec.whatwg.org/multipage/interaction.html#dom-focus-dev does seem to mention showing the keyboard?

I'm also unsure if we should call out the conditions. In theory, this feature should already work.

Comment (Manishearth) [reply index.bs:657]: Yes, but "already work" _when_? Clearly we don't expect the keyboard to be popping up randomly, nor do we expect it to show up in one specific case and not elsewhere.

Comment (cabanier) [reply index.bs]: I clarified the attribute a bit.

File: index.bs [hunk 2]
```diff
@@ -654,7 +654,7 @@ Each {{XRSession}} has a <dfn for=XRSession>set of granted features</dfn>, which
 
 The <dfn attribute for="XRSession">enabledFeatures</dfn> attribute returns the features in the [=XRSession/set of granted features=] as a new array of {{DOMString}}s.
 
-The <dfn attribute for="XRSession">isSystemKeyboardSupported</dfn> attribute indicates that the {{XRSystem}} has the ability to display the system keyboard while the {{XRSession}} is active.
+The <dfn attribute for="XRSession">isSystemKeyboardSupported</dfn> attribute indicates that the {{XRSystem}} has the ability to display the system keyboard while the {{XRSession}} is active. If {{XRSession/isSystemKeyboardSupported}} is <code>true</code>, Web APIs that would trigger the overlay keyboard (such as `focus`) will show the system keyboard. The {{XRSession}} MUST set the [=visibility state=] of the {{XRSession}} to {{XRVisibilityState/"visible-blurred"}} while the keyboard is shown.
```
Comment (Manishearth) [inline index.bs:657]: nit: link to `focus()`

Comment (cabanier) [reply index.bs]: done

**Review by Manishearth — APPROVED** [review]

**Pull request merged**